scripts/variantstore/wdl/GvsImportGenomes.wdl

```diff
@@ -16,14 +16,29 @@ workflow GvsImportGenomes {
     Boolean skip_loading_vqsr_fields = false
 
     File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
-    # If increasing this, also consider increasing `load_data_preemptible_override` and `load_data_maxretries_override`.
-    Int load_data_batch_size = 5
+    Int? load_data_batch_size
     Int? load_data_preemptible_override
     Int? load_data_maxretries_override
     File? load_data_gatk_override = "gs://broad-dsp-spec-ops/scratch/bigquery-jointcalling/jars/gg_VS-443_VETIngestValidation_20220531/gatk-package-4.2.0.0-531-gf8f4ede-SNAPSHOT-local.jar"
     String? service_account_json_path
   }
 
+  Int num_samples = length(external_sample_names)
+
+  Int effective_load_data_batch_size = if (defined(load_data_batch_size)) then select_first([load_data_batch_size])
+                                       else if (num_samples <= 100) then 1
+                                            else if (num_samples <= 5000) then 5
+                                                 else if (num_samples <= 20000) then 10
+                                                      else 20
```

Comment:
perhaps out of scope for this pr, but two suggestions:
1. let's maybe do this as a rounded fraction based on the max # of cromwell vms we feel comfortable running (rather than a step function)
2. do we want to throw an error or maybe just a warning if someone is trying to run a callset with > 20k samples and they haven't explicitly set a load_data_batch_size?

Reply:
(bwahaha this is why I want to scale test with exomes)